submit/sub.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+"""
+Submit job to CPG
+Suggested and programmed by zhili<zhilizheng@outlook.com>
+Tested and run by Zhenqiao
+License: MIT
+Please read the document from analysis-runner
+https://anaconda.org/cpg/analysis-runner
+"""
+
+from cpg_utils.hail_batch import output_path
+from cpg_utils.hail_batch import remote_tmpdir
+
+from analysis_runner.git import (
+  prepare_git_job,
+  get_repo_name_from_current_directory,
+  get_git_commit_ref_of_current_repository
+)
+
+import hailtop.batch as hb
+
+import click
+import os
+
+BILLING_PROJECT = os.getenv('HAIL_BILLING_PROJECT')
```

Comment:
These environment variables have been removed. I've suggested the new fix below: